apps/emails-and-messages/src/modules/sendgrid/configuration/sendgrid-empty-configurations.ts

```diff
@@ -0,0 +1,24 @@
+import { messageEventTypes } from "../../event-handlers/message-event-types";
+import {
+  SendgridConfiguration,
+  sendgridConfigurationEventSchema,
+  sendgridConfigurationSchema,
+} from "./sendgrid-config-schema";
+
+export const getDefaultEventsConfiguration = (): SendgridConfiguration["events"] =>
+  messageEventTypes.map((eventType) => sendgridConfigurationEventSchema.parse({ eventType }));
+
+export const getDefaultEmptyConfiguration = (): SendgridConfiguration => {
```

Comment:
can you wrap it with some `const SendgridEmptyConfiguration = {}`? We should be careful with exporting things that sound generic, but they are not (you can't use it in smtp)

Reply:
Modified export and names, hopefully its more clear now